tests/Aspire.Hosting.Redis.Tests/AddRedisTests.cs

```diff
@@ -318,6 +318,42 @@ public void WithRedisInsightSupportsChangingHostPort()
         Assert.Equal(1000, endpoint.Port);
     }
 
+    [Fact]
+    public void VerifyRedisResourceWithHostPort()
+    {
+        var builder = DistributedApplication.CreateBuilder();
+        builder.AddRedis("myredis").
+            WithHostPort(1000);
+
+        var resource = Assert.Single(builder.Resources.OfType<RedisResource>());
+        var endpoint = Assert.Single(resource.Annotations.OfType<EndpointAnnotation>());
+        Assert.Equal(1000, endpoint.Port);
+    }
+
+    [Fact]
+    public async Task VerifyRedisResourceWithPassword()
+    {
+        using var builder = TestDistributedApplicationBuilder.Create();
+
+        var password = "p@ssw0rd1";
+        var pass = builder.AddParameter("pass", password);
+        var redis = builder.
+            AddRedis("myRedis")
```

Comment:
There are various code cleanups scattered around. We typically put the `.` on the trailing line, with the method being invoked.

Along with this, there are unnecessary blank lines and other whitespace issues. Can you clean these up?